.golangci.yml

```diff
@@ -14,6 +14,8 @@ issues:
   exclude-rules:
   - linters: [errcheck]
     text: "Error return value of .((os\\.)?std(out|err)\\..*|.*Close|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv). is not checked"
+  - linters: [interfacer]
+    text: "`cfg` can be `k8s.io/apimachinery/pkg/runtime.Object`"
```

Comment:
I'm not sure how to resolve this, the func this is referring to is https://github.com/kubernetes-sigs/controller-runtime/pull/891/files#diff-091ace02b49a46f55f2586cfec89a160R39 which needs to have `cfg ControllerConfiguration` because it passes this type into `AndFrom()` in `manager.go`

Reply:
Globally for the project or leave it like this?